wrappers/nodejs/.eslintrc.json

```diff
@@ -0,0 +1,29 @@
+{
+  "extends": "google",
+  "parserOptions": {
+    "ecmaVersion": 6
+  },
+  "plugins": [
+    "header"
+  ],
+  "rules": {
+    "header/header": [2, "tools/header.js"],
+    "max-len": ["error",
+      // Following Google Javascript 100 char line limit
```

Comment:
Comment is not allowed in JSON files :)

Reply:
Oh, strange, eslint seems still working. Do you want include this change in PR #601?